packages/orbit-design-tokens/scripts/fetchBaseTokens.mjs

```diff
@@ -150,7 +151,10 @@ async function getColorTokensFromFigma() {
           } else {
             await fs.writeFile(
               path.join(OUTPUT_PATH, `${palette}.json`),
-              JSON.stringify(saveTokens(palette, tokens), null, 2),
+              prettier.format(JSON.stringify(saveTokens(palette, tokens), null, 2), {
+                ...prettier.resolveConfig.sync(""),
```

Comment:
Would it make sense to use [this function](https://github.com/kiwicom/orbit/blob/189f7c1cf6260333a663e329d7348f77d9502aef/packages/orbit-design-tokens/src/dictionary/utils/format.ts#L9)?

Reply:
That would require the function to be exported and imported here. But importing something from a `.ts` file on an `.mjs` is not straightforward, as the typescript needs to be converted to js before… So I don't think it's worth the effort. WDYT?

Reply:
Ok, I get it.

This `.ts` vs `.mjs` vs `.mts` is a bit messy. I struggle to understand it and I faced some problems with `consts` files already, forcing me to duplicate them.